src/commands.js

```diff
@@ -9,6 +9,8 @@ const executeCommand = async (text, channel, userId) => {
 
   const textWithoutAction = text.replace(action.id, "").trim();
 
+  console.log("RECEIVED", action.id);
```

Comment:
If we want to log it, then maybe let's have some more log here? :D like:

```suggestion
  console.log("[INFO] Received action: ", action.id, textWithoutAction);
```

Reply:
Agree :+1: